features/works/work_related.feature

```diff
@@ -729,3 +729,47 @@ Scenario: When a user is notified that a co-authored work has been inspired by a
   Then I should see the inspiring parent work in the beginning notes
     And I should see the translation listed on the original work
     And I should see the related work listed on the original work
+
+Scenario: Notification emails for related works are translated
+
+  Given a locale with translated emails
+    And I have related works setup
+    And the user "inspiration" enables translated emails
+    And the user "encouragement" allows co-creators
+  When I am logged in as "inspiration"
+    And I edit the work "Worldbuilding"
+    And I invite the co-author "encouragement"
+    And I press "Post"
+  Then 1 email should be delivered to "encouragement"
+    And the email should contain "The user inspiration has invited your pseud encouragement to be listed as a co-creator on the following work"
+  When the user "encouragement" accepts all co-creator requests
+    And a related work has been posted
+  Then 3 emails should be delivered
```

Comment:
For the future: You can add an "all emails have been delivered" before this step and then make it 2 in this step, since we're only really checking for the two new emails here. Then you also don't need the "last email" in the steps for `encouragement`.